pkgs/applications/misc/clipqr/default.nix

```
@@ -0,0 +1,63 @@
+{ lib
+, fetchFromGitLab
+, buildGoModule
+, copyDesktopItems
+, makeDesktopItem
+, pkg-config
+, pkgs
+}:
+
+buildGoModule rec {
+  pname = "clipqr";
+  version = "1.0.0";
+
+  src = fetchFromGitLab {
+    owner = "imatt-foss";
+    repo = "clipqr";
+    rev = "v${version}";
+    sha256 = "sha256-E90nTJtx4GOacu8M7oQBznnSQVDIZatibgKMZEpTUaQ=";
+  };
+
+  vendorSha256 = "5kAOSyVbvot4TX/XfRNe1/zZk6fa7pS1Dvn9nz11u3U=";
+
+  ldflags = [ "-s" "-w" ];
+
+  buildInputs = with pkgs; [
+    libGL
+    mesa
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXext
+    xorg.libXi
+    xorg.libXinerama
+    xorg.libXrandr
+  ];
```

```suggestion
  buildInputs = [
    libGL
    mesa
    libX11
    libXcursor
    libXext
    libXi
    libXinerama
    libXrandr
  ];
```
It would be preferable if you sort these as well.